.gitignore

```diff
@@ -17,10 +17,12 @@
 *.iml
 *.ipr
 /out/
+/bin/
 
 ### NetBeans ###
 /nbproject/private/
 /nbbuild/
 /dist/
 /nbdist/
-/.nb-gradle/
\ No newline at end of file
+/.nb-gradle/
+
```

Comment:
You might want to include the bin folder here.
You are adding this file which is absolutely unnecessary:
bin/main/application.properties

Reply:
the bin folder is already in the git ignore i guess